src/Graby.php

```diff
@@ -272,6 +284,8 @@ private function doFetchContent($url)
         $re = '/<(?!iframe)([^>\s]+)[^>]*>(?:<br \/>|&nbsp;|&thinsp;|&ensp;|&emsp;|&#8201;|&#8194;|&#8195;|\s)*<\/\1>/m';
         $html = preg_replace($re, '', $html);
 
+        $this->logger->debug('HTML after regex empty nodes stripping:', ['html' => $html]);
```

Comment:
```suggestion
        $this->logger->debug('HTML after regex empty nodes stripping', ['html' => $html]);
```

Reply:
Resolved in https://github.com/j0k3r/graby/pull/184/commits/0a7d2ca09e7a0f81d9d66ac24b4acad48990d09e